### ISSUE TITLE
User story - Project creation

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#6]
Epic: #9 

### User Story

As a **tech user**, I can **create a project** so that I can **display it in my portfolio**.

### Acceptance Criteria

- Must be able to easily navigate to where to create a project
- When creating a project, the form should validate all their inputs
- The project should appear on the user's project page
- The project record should be added in the database
- The project should be associated with the user

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [x] Written unit and/or integration tests for functionality.
- [x] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Further tasks

- [x] Set up backend to url route to /user/project page
- [x] Create UI and template for project creation page
- [x] Create a `show` page to display the project
- [x] Create project model
- [x] Set up cloudinary to manage images associated with projects

**Comment by stephendawsondev** [comment]
Adding to Sprint 1 as I've completed the other tasks

**Comment by stephendawsondev** [comment]
This was moved to Sprint 2 as I didn't have time to finish in the last sprint. 

For now, I have created the project submission page and displayed a button on the user profile. The most recent merge of #69  added the tech addition field. It took some figuring out, but I think it is an important part of the user experience.

I still need to fix the UI, add form validation and add some tests before closing this issue.
